mathesar/models.py

```diff
@@ -28,13 +28,19 @@ def __str__(self):
         return f"{self.__class__.__name__}: {self.oid}"
 
 
+_engine = None
```

Comment:
Please add a TODO comment that this needs to be removed once we have the proper fix (and also add a note to #280)